policy_definitions/update_management/policy_set_definition_qby_deploy_update_management.tmpl.json

```diff
@@ -0,0 +1,90 @@
+                }
+            },
+            "managementResourceGroup": {
+                "type": "String",
+                "metadata": {
+                    "description": "Name of the management resource group. The maintenance configuration will be deloyed there."
+                }
+            },
+            "location": {
+                "type": "String",
+                "metadata": {
+                    "description": "Location of the maintenance resources.",
+                    "strongType": "location"
+                }
+            }
+        },
+        "policyDefinitions": [
+            {
+                "policyDefinitionReferenceId": "Configure update management for virtual machines with a given tag using Azure Update Manager",
+                "policyDefinitionId": "${root_scope_resource_id}/providers/Microsoft.Authorization/policyDefinitions/QBY-Deploy-Updates",
+                "parameters": {
+                    "managementSubscriptionId": {
+                        "value": "[parameters('managementSubscriptionId')]"
+                    },
+                    "managementResourceGroup": {
+                        "value": "[parameters('managementResourceGroup')]"
+                    },
+                    "location": {
+                        "value": "[parameters('location')]"
+                    }
+                },
+                "groupNames": []
+            },
+            {
+                "policyDefinitionReferenceId": "Require a tag 'severity group monthly' exists and is not empty",
+                "policyDefinitionId": "${root_scope_resource_id}/providers/Microsoft.Authorization/policyDefinitions/QBY-Require-Severity-Group-Tag",
+                "parameters": {},
+                "groupNames": []
+            },
+            {
+                "policyDefinitionReferenceId": "Configure patch mode and bypassPlatformSafetyChecksOnUserSchedule setting of windows vm for Azure Update Manager",
+                "policyDefinitionId": "${root_scope_resource_id}/providers/Microsoft.Authorization/policyDefinitions/QBY-Set-Update-Settings",
+                "parameters": {},
+                "groupNames": []
+            },
+            {
+                "policyDefinitionReferenceId": "Configure periodic checking for missing system updates on azure virtual machines",
+                "policyDefinitionId": "/providers/Microsoft.Authorization/policyDefinitions/59efceea-0c96-497e-a4a1-4eb2290dac15",
+                "parameters": {},
+                "groupNames": []
+            },
+            {
+                "policyDefinitionReferenceId": "Configure periodic checking for missing system updates on azure Arc-enabled servers",
+                "policyDefinitionId": "/providers/Microsoft.Authorization/policyDefinitions/bfea026e-043f-4ff4-9d1b-bf301ca7ff46",
+                "parameters": {},
+                "groupNames": []
+            },
```

Comment:
These built in policies have parameters. Some of them have emtpy default values or are scoped to either windows or linux exclusively. Please check if non of these parameters need to be set and if this covers linux (in case that needs to be covered)